src/ParallelAlgorithms/Events/ObserveFields.hpp

```diff
@@ -306,7 +311,7 @@ class ObserveFields<VolumeDim, ObservationValueTag, tmpl::list<Tensors...>,
 
  private:
   std::string subfile_path_;
-  std::unordered_set<std::string> variables_to_observe_{};
+  std::unordered_set<std::string> variables_to_observe_;
```

Comment:
Keep the default-construction please. Everything has to be default-constructible for Charm++. I guess removing the braces still default-constructs this but I think we prefer being explicitly about it (and clang-tidy does, too)